.github/actions/check-previous-test-runs/action.yml

```
@@ -67,13 +67,23 @@ runs:
-          body: |
-            ✅ Previously successful run of **${{ inputs.workflow_name }}** workflow has been found.
-            ⏭️ Skipping tests for this run.
-            💡 If you are unsure about your latest changes, please rerun the workflow manually. (Use the Re-run all jobs option)
+        comment-id: ${{ steps.find-comment.outputs.comment-id }}
+        token: ${{ inputs.github_token }}
+        issue-number: ${{ github.event.pull_request.number }}
+        body: |
+          ✅ Previously successful run of **${{ inputs.workflow_name }}** workflow has been found.
+          ⏭️ Skipping tests for this run.
+          💡 If you are unsure about your latest changes, please rerun the workflow manually. (Use the Re-run all jobs option)
+        edit-mode: replace
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)